csrc/transform_replay.cpp

```diff
@@ -701,6 +701,11 @@ std::pair<TensorDomain*, int64_t> TransformReplay::replayCasP(
     used_IDs.emplace(it->second);
   }
 
+  // pay attention to consumer pos, it should be the actual replayed axis,
+  // otherwise further replay may lead to error where the mapped ID is missing.
+  // see https://github.com/NVIDIA/Fuser/issues/2593
```

Comment:
nitpick.

Totally not your problem. But can we have a follow up PR to fix the comment above at [here]( https://github.com/NVIDIA/Fuser/blob/8f974c8d39af9851bacb34af08c10287f3e9c077/csrc/transform_replay.cpp#L664-L685). I think it's copied from [here](https://github.com/NVIDIA/Fuser/blob/8f974c8d39af9851bacb34af08c10287f3e9c077/csrc/transform_replay.cpp#L433-L453), but we forgot to rename things.

Even better, would be to merge the comment in a single place and reference it in the other place.